### ISSUE TITLE
User Roles - Collection Listings: Size Issue

### ISSUE THREAD

**Issue by philleepflorence** [directus/app#2401]
Noticed that one cannot add a lot of custom collection listings as the size of the field in directus_roles->collection_listing is 255 (VARCHAR)...

A temporary fix was done by manually updating the size in the DB to 500.

On a side note, would it be possible to edit the name displayed of the collections in the menu column?

**Comment by rijkvanzanten** [comment]
Good catch, that should be `TEXT` instead of `VARCHAR` (@directus/api 🔔 ) as we're storing JSON in that field. The same goes for `module_listing`.

> On a side note, would it be possible to edit the name displayed of the collections in the menu column?

Yes! You can add custom translations for each collection from the collection settings. If a translation exists for your current language, it will use and display that.
